fala/settings/base.py

```diff
@@ -22,6 +22,7 @@
 DATABASES = {}
 
 ALLOWED_HOSTS = [
+    '.laa-fala-production.apps.cloud-platform-live-0.k8s.integration.dsd.io',
     '.laa-fala-staging.apps.cloud-platform-live-0.k8s.integration.dsd.io',
```

Comment:
`ALLOWED_HOSTS` seems to be very environment-specific. What do you think about injecting it as configuration instead? The same way we do with `LAALAA_API_HOST`?

Reply:
I'm hesitant to do this until we've migrated to Kubernetes. I don't want to figure out how to do environment variables in Template Deploy if I can avoid it. Unless you already know how?

Reply:
We could do a lookup and default to the below values `['.fala.dsd.io', '.find-legal-advice.justice.gov.uk']` if it does not exist. What do you think?

Reply:
Good idea. Will do.